src/components/submit-game/submit-game.component.html

```diff
@@ -0,0 +1,42 @@
+<mat-toolbar color="primary">
+    <span>Spiel erstellen</span>
+</mat-toolbar>
+<mat-card>
+  <mat-card-content>
+    <div class="form">
+      <h4><span class="player">Spieler</span> und <span class="playmate">Mitspieler</span> auswählen</h4>
+      <mat-button-toggle #player1 [disabled]="buttonsDisabled[0]" [ngClass]="{'player': player == 0, 'playmate': playmate == 0}" (change)="changePlayers(0, $event, player1, player2, player3, player4)" class="full-size">{{ players[0].name }}</mat-button-toggle>
+      <mat-button-toggle #player2 [disabled]="buttonsDisabled[1]" [ngClass]="{'player': player == 1, 'playmate': playmate == 1}" (change)="changePlayers(1, $event, player1, player2, player3, player4)" class="full-size">{{ players[1].name }}</mat-button-toggle>
+      <mat-button-toggle #player3 [disabled]="buttonsDisabled[2]" [ngClass]="{'player': player == 2, 'playmate': playmate == 2}" (change)="changePlayers(2, $event, player1, player2, player3, player4)" class="full-size">{{ players[2].name }}</mat-button-toggle>
+      <mat-button-toggle #player4 [disabled]="buttonsDisabled[3]" [ngClass]="{'player': player == 3, 'playmate': playmate == 3}" (change)="changePlayers(3, $event, player1, player2, player3, player4)" class="full-size">{{ players[3].name }}</mat-button-toggle><br>
+      <hr>
+      <mat-button-toggle-group #gameType="matButtonToggleGroup">
```

Comment:
I think for better user experience on web we should add some tooltips with the gametype name

Reply:
Depends on what content the buttons are going to have, please open an issue for the future

Reply:
See #11